apps/etterlatte-vedtaksvurdering/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/Samordningsdata.kt

```diff
@@ -28,3 +30,9 @@ class Samordningsmelding(
     val refusjonskrav: Boolean,
     val versjon: Long,
 )
+
+class SamordningsvedtakWrapper(
```

Comment:
Kunne du ikke bare lagt `behandlingId`i klassen du unwrapper?  

Og hvorfor ikke `data class`? Samme med `Samordningsvedtak` over

Reply:
Instansene av klassen som unwrappes kommer fra jackson. Jeg _kunne_ ha laget noe custom deserializer for å gjøre dette direkte under deserialiseringen men det fremstår unødvendig komplekst.